app/controllers/request-to-go-live/organisation-address/get.controller.js

```diff
@@ -8,12 +8,15 @@ const { response } = require('../../../utils/response')
 const { countries } = require('@govuk-pay/pay-js-commons').utils
 const formatServicePathsFor = require('../../../utils/format-service-paths-for')
 const { getAlreadySubmittedErrorPageData } = require('../../stripe-setup/stripe-setup.util')
+const { isSwitchingCredentialsRoute } = require('../../../utils/credentials')
 
 module.exports = function getOrganisationAddress (req, res) {
   const isRequestToGoLive = Object.values(paths.service.requestToGoLive).includes(req.route && req.route.path)
+  const isStripeUpdateOrgDetails = req.url ? req.url.startsWith('/your-psp/') : false
+  const isSwitchingCredentials = isSwitchingCredentialsRoute(req)
 
-  const isStripeUpdateOrgDetails = paths.account.yourPsp.stripeSetup.updateOrgDetails.includes(req.route && req.route.path)
-
+  const isStripeSetupUserJourney = isStripeUpdateOrgDetails ? true : isSwitchingCredentials ? true : false
```

Comment:
If Boolean is used above then this could be 

```suggestion
  const isStripeSetupUserJourney = isStripeUpdateOrgDetails || isSwitchingCredentials
```